### PR TITLE
NAMD parser for FEP files

### PR DESCRIPTION
**Pull request by vtlim**
This pull request parses NAMD .fepout files to extract the work values (`dE` column) into a pandas dataframe in line with the overall API proposal. I also added a few tests to check the dimensions of the imported data and verify the results of BAR analysis on the data in `alchemtest`.

### CODE REVIEW

**Comment by vtlim** [comment]
For issue #7

**Comment by orbeckst** [comment]
@vtlim sorry, been busy, and didn't see your PR. Will have a look at it, eventually (unless anyone else beats me to it). Feel free to ping me @orbeckst

File: src/alchemlyb/parsing/namd.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    df = pd.concat([df, tempDF])
+    df.fillna(0, inplace=True)
+
+    df.set_index(['timestep','fep-lambda'], inplace=True)
+    return df
+
+
+
+
+def extract_dHdl(fepout):
```
Comment (orbeckst) [inline src/alchemlyb/parsing/namd.py]: If it's not implemented then leave it out, better than a `pass` and weird failures down the line.

Comment (orbeckst) [reply src/alchemlyb/parsing/namd.py]: Also, removing it will improve your coverage ;-)

Comment (vtlim) [reply src/alchemlyb/parsing/namd.py]: Got it, I'll take out this placeholder function.

**Pull request merged**